Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -39,17 +39,25 @@ function Get-AzureUri
 function Get-ProxyUri
 {
     param([String] [Parameter(Mandatory=$true)] $serverUrl)
+    
+    $proxyUri = (Get-Item env:AGENT_PROXYURL).value
```

Comment:
@TingluoHuang to confirm, i believe the interesting variables are VSTS_HTTP_PROXY VSTS_HTTP_PROXY_USERNAME and VSTS_HTTP_PROXY_PASSWORD